Docs/Building.md

```diff
@@ -108,6 +109,7 @@ For the `OPENSSL_ROOT_DIR` parameter, set the directory for where ever the opens
 
 - macOS: `/usr/local/opt/openssl` or otherwise depending on your setup (if you're using HomeBrew, MacPorts or installed in from source)
 - Linux: This parameter is not necessary on linux. (CMake can find it on it's own)
+- > **Note for Arch Linux Users:** CMake has trouble finding `libssl` and `libcrypto` by defeault.  Specifying them with `-DOPENSSL_SSL_LIBRARY=/usr/lib/libssl.so.1.1` and `-DOPENSSL_CRYPTO_LIBRARY=/usr/lib/libcrypto.so.1.1` seems to fix the issue.
```

Comment:
`defeault` -> `default`

Reply:
Also consider rewording and/or instead using full code blocks to try to get these commands to not be split across lines.